src/test/validatorTest.ts

```diff
@@ -175,99 +175,99 @@ describe('validator', () => {
           let result = validator.validateJsonObject(input);
           expect(result).to.have.deep.property('templateValid', false);
           expect(result['errors']['crit']).to.have.lengthOf(1);
```

Comment:
Can you revert the whitespace changes?

Reply:
sure, just to keep it out of the merge commit for this pr?